Tests/Integration/Microsoft365DSC/M365DSCIntegration.AAD.Create.Tests.ps1

```diff
@@ -290,6 +290,19 @@
                     Principal       = "AdeleV@$TenantId";
                     RoleDefinition  = "Catalog creator";
                 }
+                AADFeatureRolloutPolicy 'AADFeatureRolloutPolicy-CertificateBasedAuthentication rollout policy'
```

Comment:
These tests are generated automatically based on the Example files and we shouldn't manually edit them.